addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -209,8 +209,7 @@ export class MassMailingHtmlField extends HtmlField {
         $snippetsSideBar.find(`.o_we_website_top_actions > *:not(${selectorToKeep})`).attr('style', 'display: none!important');
 
         if (device.isMobile) {
-            $snippetsSideBar.hide();
-            this.$content.attr('style', 'padding-left: 0px !important');
+            this.wysiwyg.snippetsMenu.destroy();
```

Comment:
So... how do you open it back ? Is there a button or something ? I didn't try the branch, just looking at the diff.

Reply:
I didn't find that there was a way to do it originally so I assume you just.. don't open it back. It doesn't seem like it was possible in previous versions, as far as I could tell.

Reply:
Completely forgot to respond after looking at it. It doesn't seem like there was any way before. I assume the idea was that the screen would not grow, so there would be no reason to open it again.

Reply:
I'm not sure this is correct honestly, because `device.isMobile` is, in practice, `screen.isQuiteSmall`. Consider the following scenario:

The client is using Odoo in an environment where they use, for example, two windows side by side or some other window configuration that happen to make their window quite small. It's ok, they're used to work like that. When they open the Email Marketing app however, they notice they don't have the snippets menu. Right, it's because the window is too small. It's not comfortable to edit a mailing in a small window anyway so they click on the "maximize" button of the window, hoping to get back the snippets menu. Unfortunately, it will never come back because it's been destroyed.

To be honest, I wouldn't be surprised that this use case likely wasn't supported before your changes anyway, but I think it should be. Therefore, I think destroying the component might be too radical of an action in this case. I think hiding should be done in css with media queries rather than in js so that the snippets menu can hide and reappear dynamically. What do you think @tde-banana-odoo ? It's ok for me if we decide we don't want to support the above use case, but I wanted to point it out because the naming of `isMobile` is misleading IMO.

Reply:
Makes sense, this was originally to quickly get rid of the traceback but I'm seeing that's been fixed since.

I don't mind modifying the task to add support for it, personally. But it does mean modifying some wysiwys stuff, since everything is static right now (notably I don't think device.isMobile is ever updated even is the screen gets bigger)

Reply:
Oh, if it's the case then I guess my example use case is rather wildly out of scope.

Reply:
It's 2 birds 1 stone though, I think it's fine

Reply:
Hi @dmo-odoo  :)

Could you check this out once again? Especially the approach of the last commit. 

The purpose is less to be able to fold-unfold without re-instanciating, and more that we use the snippetMenu to load some assets in mass_mailing_html_field (though I'm not sure why exactly, maybe that should change). But then we still need the snippetMenu to put back the toolbar when it's folded.